src/test/kotlin/lotto/LottoGeneratorTest.kt

```diff
@@ -0,0 +1,15 @@
+package lotto
+
+import lotto.domain.LottoGenerator
+import lotto.domain.LottoSeller
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class LottoGeneratorTest {
+    @Test
+    fun `로또 구매 갯수만큼 로또 번호를 생성한다`() {
+        val lottoSeller = LottoSeller(5000)
```

Comment:
저는 금액을 노출할 때에는 가독성을 위해 다음과 같이 표현합니다. 참고만 해주셔도 좋습니다.
```suggestion
        val lottoSeller = LottoSeller(5_000)
```